### PR TITLE
br: change the compatiblity table

### PR DESCRIPTION
**Pull request by 3pointer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/backup-and-restore-overview.md [hunk 1]
```diff
@@ -121,17 +121,17 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
...
-| TiDB v6.5 PITR 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 | 兼容 |
+在执行备份和恢复操作前，BR 会执行自身版本与 TiDB 集群版本的对比检查。如果版本之间不兼容，BR 会提示报错并退出。如要忽略版本检查，可以通过设置 `--check-requirements=false` 强行跳过版本检查。需要注意的是，跳过检查可能会遇到数据不兼容的问题。
+
+| 恢复版本（横向）\ 备份版本（纵向）   | 恢复到 TiDB v6.0 | 恢复到 TiDB v6.1| 恢复到 TiDB v6.2 | 恢复到 TiDB v6.3,v6.4,v6.5 | 恢复到 TiDB v6.6 |
+|  ----  |  ----  | ---- | ---- | ---- | ---- |
+| TiDB v6.0,v6.1,v6.2,v6.3,v6.4 快照备份 | 兼容（已知问题，如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)） | 兼容 | 兼容 | 兼容 | 兼容(需使用 v6.6 及以上版本的 BR) |
+| TiDB v6.3,v6.4,v6.5 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 |
+| TiDB v6.6 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 | 
+
+> **注意：**
```
Comment (3pointer) [inline br/backup-and-restore-overview.md:132]: 可以挪到表格上面

File: br/backup-and-restore-overview.md [hunk 2]
```diff
@@ -121,17 +121,17 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
...
-| 恢复版本（横向）\ 备份版本（纵向）   | 恢复到 TiDB v6.0 | 恢复到 TiDB v6.1| 恢复到 TiDB v6.2 | 恢复到 TiDB v6.3 | 恢复到 TiDB v6.4 | 恢复到 TiDB v6.5 |
-|  ----  |  ----  | ---- | ---- | ---- | ---- | ---- |
-| TiDB v6.0 快照 | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.1 快照备份 | 兼容（已知问题，如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)） | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.2 快照备份 | 兼容（已知问题，如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)） | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.3 快照备份 | 兼容（已知问题，如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)） | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.3 PITR 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.4 PITR 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.5 PITR 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 | 兼容 |
+在执行备份和恢复操作前，BR 会执行自身版本与 TiDB 集群版本的对比检查。如果版本之间不兼容，BR 会提示报错并退出。如要忽略版本检查，可以通过设置 `--check-requirements=false` 强行跳过版本检查。需要注意的是，跳过检查可能会遇到数据不兼容的问题。
```
Comment (shichun-0415) [inline br/backup-and-restore-overview.md:124]:
```suggestion
在执行备份和恢复操作前，BR 会比较自身版本与 TiDB 集群版本。如果版本之间不兼容，BR 会提示报错并退出。如要忽略版本检查，可以通过设置 `--check-requirements=false` 强行跳过版本检查。需要注意的是，跳过检查可能会遇到数据不兼容的问题。
```

File: br/backup-and-restore-overview.md [hunk 3]
```diff
@@ -121,17 +121,17 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
...
+
+| 恢复版本（横向）\ 备份版本（纵向）   | 恢复到 TiDB v6.0 | 恢复到 TiDB v6.1| 恢复到 TiDB v6.2 | 恢复到 TiDB v6.3,v6.4,v6.5 | 恢复到 TiDB v6.6 |
+|  ----  |  ----  | ---- | ---- | ---- | ---- |
+| TiDB v6.0,v6.1,v6.2,v6.3,v6.4 快照备份 | 兼容（已知问题，如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)） | 兼容 | 兼容 | 兼容 | 兼容(需使用 v6.6 及以上版本的 BR) |
```
Comment (shichun-0415) [inline br/backup-and-restore-overview.md:128]:
```suggestion
| TiDB v6.0、v6.1、v6.2、v6.3 及 v6.4 快照备份 | 兼容（已知问题，如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)） | 兼容 | 兼容 | 兼容 | 兼容（需使用 v6.6 及以上版本的 BR） |
```

File: br/backup-and-restore-overview.md [hunk 4]
```diff
@@ -121,17 +121,17 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
...
+| TiDB v6.3,v6.4,v6.5 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 |
+| TiDB v6.6 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 |
```
Comment (shichun-0415) [inline br/backup-and-restore-overview.md:130]:
```suggestion
| TiDB v6.3,v6.4,v6.5 日志备份| 不兼容 | 不兼容 | 不兼容 | 兼容 | 兼容 |
| TiDB v6.6 日志备份| 不兼容 | 不兼容 | 不兼容 | 兼容 | 兼容 | 
```

File: br/backup-and-restore-overview.md [hunk 5]
```diff
@@ -121,17 +121,17 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
...
-| TiDB v6.3 快照备份 | 兼容（已知问题，如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)） | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.3 PITR 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.4 PITR 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.5 PITR 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 | 兼容 |
+在执行备份和恢复操作前，BR 会执行自身版本与 TiDB 集群版本的对比检查。如果版本之间不兼容，BR 会提示报错并退出。如要忽略版本检查，可以通过设置 `--check-requirements=false` 强行跳过版本检查。需要注意的是，跳过检查可能会遇到数据不兼容的问题。
+
+| 恢复版本（横向）\ 备份版本（纵向）   | 恢复到 TiDB v6.0 | 恢复到 TiDB v6.1| 恢复到 TiDB v6.2 | 恢复到 TiDB v6.3,v6.4,v6.5 | 恢复到 TiDB v6.6 |
+|  ----  |  ----  | ---- | ---- | ---- | ---- |
+| TiDB v6.0,v6.1,v6.2,v6.3,v6.4 快照备份 | 兼容（已知问题，如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)） | 兼容 | 兼容 | 兼容 | 兼容(需使用 v6.6 及以上版本的 BR) |
+| TiDB v6.3,v6.4,v6.5 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 |
```
Comment (hfxsd) [inline br/backup-and-restore-overview.md:129]:
```suggestion
| 恢复版本（横向）\ 备份版本（纵向）   | 恢复到 TiDB v6.0 | 恢复到 TiDB v6.1| 恢复到 TiDB v6.2 | 恢复到 TiDB v6.3, v6.4, v6.5 | 恢复到 TiDB v6.6 |
|  ----  |  ----  | ---- | ---- | ---- | ---- |
| TiDB v6.0, v6.1, v6.2, v6.3, v6.4 快照备份 | 兼容（已知问题，如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)） | 兼容 | 兼容 | 兼容 | 兼容 （需使用 v6.6 及以上版本的 BR） |
| TiDB v6.3, v6.4, v6.5 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 |
```

File: br/backup-and-restore-overview.md [hunk 6]
```diff
@@ -121,17 +121,17 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
...
+| TiDB v6.6 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 | 
+
+> **注意：**
+> 
+> 在执行备份和恢复操作时，建议使用的 BR 版本号与 TiDB 集群版本号相同。
```
Comment (shichun-0415) [inline br/backup-and-restore-overview.md:134]:
```suggestion
> 在执行备份和恢复操作时，建议使用相同版本的 BR 和 TiDB 集群。
```

File: br/backup-and-restore-overview.md (hunk 2 above)
Comment (hfxsd) [inline br/backup-and-restore-overview.md:124]: `--check-requirements=false`  可以加个链接

File: br/backup-and-restore-overview.md [hunk 7]
```diff
@@ -121,17 +121,17 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
...
-| TiDB v6.0 快照 | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.1 快照备份 | 兼容（已知问题，如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)） | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.2 快照备份 | 兼容（已知问题，如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)） | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.3 快照备份 | 兼容（已知问题，如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)） | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.3 PITR 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.4 PITR 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.5 PITR 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 | 兼容 |
+在执行备份和恢复操作前，BR 会执行自身版本与 TiDB 集群版本的对比检查。如果版本之间不兼容，BR 会提示报错并退出。如要忽略版本检查，可以通过设置 `--check-requirements=false` 强行跳过版本检查。需要注意的是，跳过检查可能会遇到数据不兼容的问题。
+
+| 恢复版本（横向）\ 备份版本（纵向）   | 恢复到 TiDB v6.0 | 恢复到 TiDB v6.1| 恢复到 TiDB v6.2 | 恢复到 TiDB v6.3,v6.4,v6.5 | 恢复到 TiDB v6.6 |
```
Comment (shichun-0415) [inline br/backup-and-restore-overview.md:126]:
```suggestion
| 备份版本（纵向）\ 恢复版本（横向）  | 恢复到 TiDB v6.0 | 恢复到 TiDB v6.1| 恢复到 TiDB v6.2 | 恢复到 TiDB v6.3,v6.4,v6.5 | 恢复到 TiDB v6.6 |
```

File: br/backup-and-restore-overview.md [hunk 8]
```diff
@@ -121,17 +121,17 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
...
-| TiDB v6.0 快照 | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.1 快照备份 | 兼容（已知问题，如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)） | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.2 快照备份 | 兼容（已知问题，如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)） | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.3 快照备份 | 兼容（已知问题，如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)） | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.3 PITR 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.4 PITR 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.5 PITR 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 | 兼容 |
+> **注意：**
+>
+> 在执行备份和恢复操作时，建议使用相同版本的 BR 和 TiDB 集群。
```
Comment (3pointer) [inline br/backup-and-restore-overview.md:126]:
```suggestion
> 在执行备份和恢复操作时，建议使用和TiDB 集群相同版本的 BR 。
```

File: br/backup-and-restore-overview.md (hunk 2 above)
Comment (shichun-0415) [reply br/backup-and-restore-overview.md]: BR 似乎没有专门的文档介绍这个配置项，是否可以不加链接呢？ @3pointer

File: br/backup-and-restore-overview.md [hunk 9]
```diff
@@ -121,17 +121,17 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
...
-| TiDB v6.0 快照 | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.1 快照备份 | 兼容（已知问题，如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)） | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.2 快照备份 | 兼容（已知问题，如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)） | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.3 快照备份 | 兼容（已知问题，如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)） | 兼容 | 兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.3 PITR 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.4 PITR 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 | 兼容 |
-| TiDB v6.5 PITR 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 | 兼容 |
+> **注意：**
+>
+> 在执行备份和恢复操作时，建议使用和 TiDB 集群相同版本的 BR 。
```
Comment (Oreoxmt) [inline br/backup-and-restore-overview.md:126]:
```suggestion
> 在执行备份和恢复操作时，建议使用和 TiDB 集群相同版本的 BR。
```

File: br/backup-and-restore-overview.md [hunk 10]
```diff
@@ -121,17 +121,17 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
...
+
+在执行备份和恢复操作前，BR 会比较自身版本与 TiDB 集群版本。如果版本之间不兼容，BR 会提示报错并退出。如要忽略版本检查，可以通过设置 `--check-requirements=false` 强行跳过版本检查。需要注意的是，跳过检查可能会遇到数据不兼容的问题。
```
Comment (Oreoxmt) [inline br/backup-and-restore-overview.md:128]:
```suggestion
在执行备份和恢复操作前，BR 会比较自身版本与 TiDB 集群版本。如果版本之间不兼容，BR 会提示报错并退出。如要需要跳过版本检查，可以设置 `--check-requirements=false`。需要注意的是，跳过检查可能会遇到数据不兼容的问题。
```

File: br/backup-and-restore-overview.md [hunk 11]
```diff
@@ -121,17 +121,17 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
...
+
+| 备份版本（纵向）\ 恢复版本（横向）  | 恢复到 TiDB v6.0 | 恢复到 TiDB v6.1| 恢复到 TiDB v6.2 | 恢复到 TiDB v6.3,v6.4,v6.5 | 恢复到 TiDB v6.6 |
```
Comment (Oreoxmt) [inline br/backup-and-restore-overview.md:130]:
```suggestion
| 备份版本（纵向）\ 恢复版本（横向）  | 恢复到 TiDB v6.0 | 恢复到 TiDB v6.1| 恢复到 TiDB v6.2 | 恢复到 TiDB v6.3、v6.4、v6.5 | 恢复到 TiDB v6.6 |
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Review by joccau — APPROVED** [review]
LGTM

File: br/backup-and-restore-overview.md [hunk 12]
```diff
@@ -121,17 +121,16 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
...
-| TiDB v6.5 PITR 日志备份| \ | \ | 不兼容 | 兼容 | 兼容 | 兼容 |
+> **注意：**
+>
+> 在执行备份和恢复操作时，建议使用和 TiDB 集群相同版本的 BR。
+
+在执行备份和恢复操作前，BR 会比较自身版本与 TiDB 集群版本。如果版本之间不兼容，BR 会提示报错并退出。如要跳过版本检查，可以设置 `--check-requirements=false`。需要注意的是，跳过检查可能会遇到数据不兼容的问题。
+
+| 备份版本（纵向）\ 恢复版本（横向）  | 恢复到 TiDB v6.0 | 恢复到 TiDB v6.1| 恢复到 TiDB v6.2 | 恢复到 TiDB v6.3、v6.4 或 v6.5 | 恢复到 TiDB v6.6 |
+|  ----  |  ----  | ---- | ---- | ---- | ---- |
+| TiDB v6.0、v6.1、v6.2、v6.3 或 v6.4 快照备份 | 兼容（已知问题，如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)） | 兼容 | 兼容 | 兼容 | 兼容（需使用 v6.6 及以上版本的 BR） |
```
Comment (3pointer) [inline br/backup-and-restore-overview.md:132]:
```suggestion
| TiDB v6.0、v6.1、v6.2、v6.3、v6.4 或 v6.5 快照备份 | 兼容（已知问题，如果备份数据中包含空库可能导致报错，参考 [#36379](https://github.com/pingcap/tidb/issues/36379)） | 兼容 | 兼容 | 兼容 | 兼容（需使用 v6.6 及以上版本的 BR） |
```

**Pull request merged**